.github/workflows/build_and_test.yml

```diff
@@ -428,3 +428,41 @@ jobs:
     - name: Build with SBT
       run: |
         ./build/sbt -Pyarn -Pmesos -Pkubernetes -Phive -Phive-thriftserver -Phadoop-cloud -Pkinesis-asl -Phadoop-2.7 compile test:compile
+
+  tpcds-1g:
+    name: Run TPC-DS queries with SF=1
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout Spark repository
```

Comment:
Maybe we should do the caches here as we do above "Cache Coursier local repository" since it builds and runs the test with downloading new dependencies in any event.